docs/contributing/feature-stages.md

```diff
@@ -1,6 +1,9 @@
 # Feature stages
 
-Some Coder features are released as Alpha or Experimental.
+Some Coder features are released in feature stages before they are generally
+available.
+
+For more information about a specific feature, please contact _X_.
```

Comment:
May not need this as they're already in the docs, to learn more about X feature they can just `ctrl + k`. 

If you want to make this a community CTA, I'd say something to the effect of:

> You can consult our Github Releases for timelines on rollout for specific features, or reach out with questions in our Discord. 


The Github issues CTA under alpha would also work here, just brainstorming.

Reply:
++ I moved it and then realized it belongs in every section (which @bpmct also highlights in https://github.com/coder/coder/pull/14786#issuecomment-2374223777)

I'll move it to the top and remove the "support" H3s